packages/app-desktop/gui/Button/Button.tsx

```diff
@@ -18,28 +18,21 @@ export enum ButtonSize {
 	Normal = 2,
 }
 
-interface Props {
+type ReactButtonProps = React.DetailedHTMLProps<React.HTMLAttributes<HTMLButtonElement>, HTMLButtonElement>;
+interface Props extends Omit<ReactButtonProps, 'onClick'> {
```

Comment:
Most of the changes to the `Button` component are to pass unused props through to the `<button>` component it wraps. This removes the need to enumerate all `aria-*` props that can be used with custom `<Button>`s.